src/schema.ts

```diff
@@ -37,9 +50,21 @@ export const resolvers = {
   },
 
   Mutation: {
-    createUser: async (_, userData: CreateUserMutation): Promise<User> => {
-      const user = new User();
+    createUser: async (_, userData: CreateUserMutation, context: Context): Promise<User> => {
+      try {
+        const secret = process.env.JWT_SECRET ?? 'secret';
+        const payload: { email: string } = jwt.verify(context.token, secret) as any;
+
+        const userRepository = await getRepository(User);
+        const db_user = await userRepository.findOne({ email: payload.email });
+        if (!db_user) {
```

Comment:
```suggestion
        const dbUser = await userRepository.findOne({ email: payload.email });
        if (!dbUser) {
```

Só pra manter o padrão camelCase